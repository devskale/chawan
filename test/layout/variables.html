<style>
a { --purple: purple; color: var(--purple) }
div { color: white; --pink: yellow !important; background-color: var(--purple) }
#test {
	--test: var(--test2);
	--test2: var(--test);
	--test3: var(--test4);
	--test4: red;
	--test5: var(--test6,var(--test7));
	--test7: var(--test6,var(--test5));
	color: var(--test);
	color: var(--test3);
	background-color: var(--test5);
}
:root { --pink: pink }
a { --pink0: var(--pink); --gray: gray }
#test2 { color: var(--pink0); --pink: yellow !important; background-color: var(--gray) }
</style>
<a>
<div>purple</div>
</a>
<span id=test>test</span>
<a>
<div id=test2>test2</div>
</a>
<div style="--asdf: ">
<div style="--red: red; background: var(--asdf) var(--red)">
test
</div>
</div>
