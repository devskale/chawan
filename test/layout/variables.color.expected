[38;2;255;255;255m[48;2;128;0;128mpurple[39m                                                                          [49m
[38;2;255;40;40mtest[39m 
[38;2;255;224;234m[48;2;128;128;128mtest2[39m                                                                           [49m
